README.md

```diff
@@ -1,4 +1,54 @@
 # BSC-MEV-Sentry
 
-BSC-MEV-Sentry is a proxy service within the BSC MEV architecture designed to shield the validator network.
-It functions to forward requests and pay to builders.
+BSC-MEV-Sentry serves as the proxy service for BSC MEV architecture, It has the following features:
+
+1. Forward RPC requests under mev namespace to validators/builders.
+2. Pay builders for their bid.
```

Comment:
==> `Pay builders on behalf of validators for their bids.`

Reply:
done